### DISCUSSION TITLE
Internal contributions from Microsoft employees?

### DISCUSSION THREAD

**Discussion by cjsut** [microsoft/fluentui#32427] [Q&A] [answered]
Hi folks,

My team at Microsoft would like to contribute to parts of Fluent UI.  Are there guidelines for internal contributions?

Our Microsoft-managed accounts are unable to contribute work items, discussions, or even fork the repository:
![image](https://github.com/user-attachments/assets/2b72a554-4231-4f4a-85fe-153276915fca)

Should we use our personal GitHub accounts? Is there some GitHub organization we need to join? Any and all guidance on this point would be much appreciated!

Thank you.

**Comment by khmakoto** [comment, marked as answer]
Hi @insaneinside! You need to use a non-Microsoft EMU account that you can link to Microsoft OSS. We can chat separately about that process if you want.
